src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball(Color color, int number) {
+        this.setColor(color);
+        this.setNumber(number);
+    }
+
+    @Override
+    public String toString() {

```

Comment:
It would be better to place `toString` at the end of file. It looks more 'beautiful'.